pkg/convert/workflow.go

```diff
@@ -0,0 +1,128 @@
+package convert
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/tinkerbell/tink/pkg/apis/core/v1alpha1"
+	protoworkflow "github.com/tinkerbell/tink/protos/workflow"
+	"github.com/tinkerbell/tink/workflow"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func WorkflowYAMLToCRD(wf *workflow.Workflow, crd *v1alpha1.Workflow) {
```

Comment:
The signature of this function is different from the other convert functions which _return_ the target type.

I see that `WorkflowYAMLToCRD` is being used to update the Status of existing CRD objects elsewhere.
Perhaps `WorkflowYAMLToStatus` would be more succinct? I doubt you would want to copy over a new Status, so I imagine you will want to add another function that merges these known Status changes with an existing CRD (or Status) object.

Changing the signature to `func WorkflowYAMLToStatus(wf *workflow.Workflow)  *v1alpha1.WorkflowStatus` would also get past my other gripe with this function, that a `nil` `crd` value is ignored and returned without logging/erroring.

Reply:
I'm good to change this to `func WorkflowYAMLToStatus(wf *workflow.Workflow) *v1alpha1.WorkflowStatus`